paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -221,43 +245,45 @@ internal class DefaultFlowController @Inject internal constructor(
     }
 
     override fun presentPaymentOptions() {
-        val state = runCatching {
-            requireNotNull(viewModel.state)
-        }.getOrElse {
-            error(
-                "FlowController must be successfully initialized using " +
-                    "configureWithPaymentIntent() or configureWithSetupIntent() " +
-                    "before calling presentPaymentOptions()"
+        withValidConfiguration { state ->
```

Comment:
Should we also prevent the user from calling configure when the options are presented? Or a confirm is in flight?